test/test_CellArray.jl

```diff
@@ -53,64 +62,64 @@ mutable struct MyMutableFieldArray{T} <: FieldArray{Tuple{2}, T, 1}
     yxxx::T
 end
 
-
 @testset "$(basename(@__FILE__))" begin
     @testset "1. CellArray allocation ($array_type arrays)" for (array_type, Array, CellArray) in zip(array_types, ArrayConstructors, CellArrayConstructors)
+        Float = array_type === "Metal" ? Float32 : Float64
```

Comment:
This can actually be included in the for loop above:
 @testset "1. CellArray allocation ($array_type arrays)" for (array_type, Array, CellArray, **Float**) in zip(array_types, ArrayConstructors, CellArrayConstructors, **FloatDefaultTypes**)

And at the beginning `FloatDefaultTypes` has to be created in analogy

Reply:
Done!